frontend/src/store.ts

```diff
@@ -2214,6 +2214,48 @@ export function createStore(web3: Web3) {
         await dispatch('fetchStakeDetails', { stakeType });
       },
 
+      async stakeNfts({ state, dispatch }, { ids, stakeType }: { ids: string[], stakeType: StakeType }) {
+        const { StakingRewards, StakingToken } = getStakingContracts(state.contracts(), stakeType);
+        if(!StakingRewards || !StakingToken) return;
```

Comment:
I think it should have state.defaultAccount check, but not sure, we'll have to look into other methods.